public/data/KendallDoesCoding.json

```diff
@@ -22,18 +32,19 @@
     {
       "name": "Quiz",
       "url": "https://kendalldoescoding.gq/quiz",
-      "icon": "globe"
+      "icon": "link"
     },
     {
       "name": "Look at my subdomains/projects",
       "url": "https://kendalldoescoding.gq/subdomains",
       "icon": "globe"
-    }
+    },
```

Comment:
Remove trailing comma

Reply:
I think this is fixed in [dc12210](https://github.com/EddieHubCommunity/LinkFree/pull/1067/commits/dc122101b7e90058974c89d6ef0ec751b00034a1) but waiting not sure